spec/interactor/organizer_spec.rb

```diff
@@ -41,13 +41,15 @@ module Interactor
       before do
         allow(instance).to receive(:context) { context }
         allow(organizer).to receive(:organized) {
-          [interactor2, interactor3, interactor4]
+          [interactor2, :mapping1, interactor3, :mapping2, interactor4]
```

Comment:
@hedgesky, building on [my earlier comment](https://github.com/collectiveidea/interactor/pull/137#issuecomment-296152774), I think your use case would be well served by an `:after` callback option.